addons/binding/org.openhab.binding.knx/README.md

```diff
@@ -0,0 +1,115 @@
+
+# KNX Binding
+
+The openHAB KNX binding allows to connect to [KNX Home Automation](http://www.knx.org/) installations. Switching lights on and off, activating your roller shutters or changing room temperatures are only some examples.
+
+To access your KNX bus you either need an KNX IP gateway (like e.g. the [Gira KNX IP Router](http://www.gira.com/en/gebaeudetechnik/systeme/knx-eib_system/knx-produkte/systemgeraete/knx-ip-router.html)) or a PC running [KNXD](https://github.com/knxd/knxd) (free open source component that enables communication with the KNX bus), which is the successor of the no longer maintained [EIBD](http://www.auto.tuwien.ac.at/~mkoegler/index.php/eibd).
+
+## Supported Things
+
+The KNX binding supports two kinds of Bridges, and two kinds of Things to access actors on the KNX bus. There is an *ip* bridge to connect to KNX IP Gateways, and a *serial* bridge for connection over a serial port to a host-attached gateway. The configuration of the two kinds of bridges is very similar (see below).
+
+## Binding Configuration
+
+The binding itself does not require any special configuration. 
+
+## Bridge Configuration
+
+### IP Gateway
+
+The IP Gateway is the most commonly used way to connect to the KNX bus. At its base, the *ip* bridge accepts the following configuration parameters:
+
+|Name|Required|Description|Default value|
```

Comment:
You would make me happy by formatting the tables in the MD file as well :-D

http://www.tablesgenerator.com/markdown_tables

Select 'File' > 'Past table data ...' and past the unformatted mark down table, you can then copy paste the formatted table 👍